### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by alekseysidorov**
Just update logger version.

- Not a breaking change
- No need in updating docs
- No need in testing

### CODE REVIEW

**Review by vitvakatu — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -18,8 +18,7 @@ travis-ci = { repository = "exonum/exonum-configuration" }
 exonum = "0.4.0"
 clap = "2.25.0"
 tempdir = "0.3.5"
-env_logger = "0.4.3"
-log = "0.3.8"
+log = "0.4.0"
```
Comment (vitvakatu) [inline Cargo.toml]: Wait, why it's not 0.41?

**Review by vitvakatu — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (alekseysidorov) [reply Cargo.toml]: So it happened.

File: Cargo.toml [hunk 2]
```diff
@@ -15,11 +15,10 @@ description = "An Exonum service that provides functionality of modifying the gl
 travis-ci = { repository = "exonum/exonum-configuration" }
 
 [dependencies]
-exonum = "0.4.0"
+exonum = "0.5.0"
```
Comment (vitvakatu) [inline Cargo.toml]: Do you want to make an actual update later?

**Review by vitvakatu — APPROVED** [review]

**Comment by DarkEld3r** [comment]
Closed in favor of #82.

**Pull request closed without merging**